tests/security_tests.py

```diff
@@ -594,11 +594,16 @@ def test_is_admin_only(self):
             )
         )
 
-        self.assertTrue(
-            security_manager._is_admin_only(
-                security_manager.find_permission_view_menu("can_delete", "DatabaseView")
+        log_permissions = ["can_list", "can_show", "can_add", "can_edit", "can_delete"]
```

Comment:
By default, I think we don't want logs to be editable. I'd remove `can_add`, `can_edit`, and `can_delete` for all users.

Reply:
Right! but that is being handled on #8960. I would merge #8960 first then on the solution found adapt this change to it, or merge this and then make the change on #8960

Reply:
it's merged!

Reply:
OK, works for me.